crypto/build.gradle

```diff
@@ -1,6 +1,5 @@
-apply plugin: 'com.android.library'
-apply plugin: 'kotlin-android'
-apply from: '../buildsystem/coverageReport.gradle'
```

Comment:
We need to enable this again. This gradle task probably depends on some android specifics so we need to check how to adjust this.